x-pack/plugins/index_lifecycle_management/public/sections/components/learn_more_link.js

```diff
@@ -7,11 +7,11 @@
 import React from 'react';
 import { EuiLink } from '@elastic/eui';
 import { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } from 'ui/documentation_links';
-import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
-const esBase = `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${DOC_LINK_VERSION}/`;
+import { FormattedMessage } from '@kbn/i18n/react';
 
+const esBase = `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${DOC_LINK_VERSION}/`;
 
-export class LearnMoreLinkUi extends React.PureComponent {
+export class LearnMoreLink extends React.PureComponent {
```

Comment:
TODO: Confirm this looks correct in the UI.

Reply:
![image](https://user-images.githubusercontent.com/1238659/59351447-4da29700-8cd3-11e9-986c-7642634dc41a.png)